### PR TITLE
🐛 Fix flaking test template

### PR DESCRIPTION
**Pull request by mboukhalfa**
Yaml indentation issue

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/flaking-test.yaml [hunk 1]
```diff
@@ -39,10 +39,10 @@ body:
...
- - type: textarea
-    id: templateLabel
-    attributes:
-      label: Label(s) to be applied
-      value: |
-        /kind flake
-        One or more /area label. See https://github.com/metal3-io/cluster-api-provider-metal3/labels for the list of labels.
\ No newline at end of file
+  - type: textarea
+      id: templateLabel
```
Comment (furkatgofurov7) [inline .github/ISSUE_TEMPLATE/flaking-test.yaml:43]: there seems to be still indentation issue, basically `id`and whole thing belopw should be aligned on the same line with `type`

**Review by furkatgofurov7 — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/flaking-test.yaml (hunk 1 above)
Comment (mboukhalfa) [reply .github/ISSUE_TEMPLATE/flaking-test.yaml]: Nice catch

**Review by Sunnatillo — COMMENTED** [review]
/lgtm

**Review by furkatgofurov7 — APPROVED** [review]

**Pull request merged**